### ISSUE TITLE
Referencing sections instead of pages in the bibliograph (backref=section)

### ISSUE THREAD

**Issue by martenlienen** [plk/biblatex#1282]
Hyperref has an option where it creates backreferences to the sections instead of pages. Since in papers sections are often shorter than a page, those would be more accurate and helpful backreferences than pages. Would it be possible to add that option to biblatex?

**Comment by moewew** [comment]
I've had a quick look at the implementation in `backref.sty`. As far as I can tell the feature actually just links to the last label, which is usually a section-like object. That means that `backref=section,` would actually reference any of `\chapter`, `\section`, `\subsection` etc. That's probably not an issue, but it's something to keep in mind. A bigger issue is that the link does not always go to the right object.

Take
```latex
\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[backref]{hyperref}

\begin{document}
\section{Foo}
Lorem \cite{lorem}
ipsum \cite{ipsum}
\begin{equation}
  xxx
\end{equation}

\subsection{Bar}
ipsum \cite{lorem}
\begin{equation}
  xxx
\end{equation}
dolor \cite{ipsum}

\begin{thebibliography}{2}

\bibitem{lorem} Lorem

\bibitem{ipsum} Ipsum

\end{thebibliography}
\end{document}
```
The backlink for "1.1" in the second reference entry actually goes to the equation and not the section heading.

---

I don't think it would be easy to implement `backref` in a more consistent way. But I'm not sure if it's a good idea to add the feature in this state, either.

**Comment by martenlienen** [comment]
You are right, I just saw the wrong reference targets in a paper of mine where I used hyperref's `backref`. If you think that it is not possible or too much work to implement, should we close this issue?

**Comment by moewew** [comment]
Let's leave it open for now. Maybe someone has an idea for a more stable implementation. But if nothing comes of it, we may close this issue when we next clean up the issues after a release.
